### PR TITLE
New "end game" event added

### LINKED ISSUE #81: Check, if game ends
Output message "Player ... wins"

### PR DESCRIPTION
**Pull request by bronvic**
It sends when there are one player on the field left
Frontend shows alert on this event

Closes #81

### CODE REVIEW

File: backend/src/game_server.rs [hunk 1]
```diff
@@ -247,12 +247,29 @@ impl GameServer {
 
         self.deselect_unit();
 
-        self.send_current_player(State::new(STATE_WAIT.to_string()));
+        if self.game.ends() {
```
Comment (bronvic) [inline backend/src/game_server.rs:250]: you may notice that this solution is quite ugly

there was an idea about wrapping response messages into enum and choose one of enum instances in condition branch, but it requires quite a big refactoring

i think all system with sending messages should be refactor with this strange communicator.rs

Comment (Hairash) [reply backend/src/game_server.rs:250]: Better to write 3 lines of code instead of function

**Review by Hairash — DISMISSED** [review]
Here we go!

**Review by Hairash — APPROVED** [review]
We are number one!

**Pull request merged**